README.rst

```diff
@@ -13,38 +13,20 @@ domain-specific language (DSL).
    :alt:
 
 ⚡️ Quick Start
---------------
+---------------
```

Comment:
Looks wrong, but probably the emoji is 2 characters, pypi complains on upload...

Reply:
:+1: